networkx/generators/tests/test_inverse_line.py

```diff
@@ -0,0 +1,20 @@
+import networkx as nx
+from nose.tools import *
+
+from networkx.testing.utils import *
+# TODO
+# JC: What other tests would we like to see here?
```

Comment:
You can also add test to increase the coverage of your new functions: https://coveralls.io/builds/7739289/source?filename=networkx%2Fgenerators%2Finverse_line.py (red lines are not used by the test suite).